special-pages/pages/new-tab/app/types.js

```diff
@@ -41,3 +41,4 @@ export const useTelemetry = () => useContext(TelemetryContext);
 
 export const InitialSetupContext = createContext(/** @type {InitialSetupResponse} */ ({}));
 export const useInitialSetupData = () => useContext(InitialSetupContext);
+export const useSettings = () => useContext(InitialSetupContext);
```

Comment:
Why do we have `useInitialSetupData` and `useSettings` doing seemingly the same thing?